code/game/gamemodes/wizard/artefact.dm

```diff
@@ -88,6 +88,10 @@
 	activate_descriptor = "depression"
 	rend_desc = "Gently wafting with the sounds of endless laughter."
 	icon_state = "clownrender"
+/obj/item/weapon/veilrender/honkrender/attack(mob/living/carbon/M, mob/living/carbon/user)
+	var/honknoise = 'sound/items/bikehorn.ogg'
```

Comment:
Also, it wouldnt work within the attack proc, it has to be defined with the description and everything else